src/Illuminate/Http/Client/RequestException.php

```diff
@@ -26,6 +33,37 @@ public function __construct(Response $response)
         $this->response = $response;
     }
 
+    /**
+     * Enable truncation of the exception message.
+     *
+     * @return void
+     */
+    public static function truncate()
+    {
+        static::$truncateAt = 120;
+    }
+
+    /**
+     * Disable truncation of the exception message.
+     *
+     * @return void
+     */
+    public static function dontTruncate()
+    {
+        static::$truncateAt = false;
```

Comment:
```suggestion
        static::$truncateAt = 0;
```

force as `int` instead of `int|false`

Reply:
This is counterintuitive as this implies truncating the message entirely instead of not truncating at all.

Reply:
what do you think about `-1`?

Reply:
Slightly better, yet I prefer to use a distinct type to indicate a behavioral flag.